config/routes.rb

```diff
@@ -1,25 +1,63 @@
-Rails.application.routes.draw do
+Rails.application.routes.draw do  
+  # PART 1: EACH IN ERB
+  # ===================
 
-  get("/", { :controller => "fortunes", :action => "lucky"})
+  # Let users visit URLs:
+  # - /lottery/lucky
+  # - /lottery/unlucky
 
-  get("/lucky-numbers", { :controller => "fortunes", :action => "lucky"})
-  get("/unlucky-numbers", { :controller => "fortunes", :action => "unlucky"})
-
-  # PART 1: STATIC ROUTES
-  # =====================
+  # PART 2: RCAV DEBUGGING
+  # ======================
 
   # Uncomment each route below ONE AT A TIME and debug.
+  # Do NOT uncomment more than one at a time, or you'll be dealing with multiple syntax errors at once.
+
+  # get("/zodiacs/aries", { :controller =>  fire, action =>  "ram" })
+  # get("/zodiacs/leo", { :controller => "fire", :action => "lion" })
+  # get("/zodiacs/sagittarius" { :controller => "fire", :action => "archer" })
+  
+  # get("/zodiacs/taurus", { :controller, "earth", :action, "bull" })
+  # get("/zodiacs/virgo", { :controller => "earth", :action => "maiden" })
+  # get("/zodiacs/capricorn", { :controller => "earth", :action => "goat" })
+  
+  # get("/zodiacs/gemini", { :controller => "air", :action => "twins" })
+  # get(" /zodiacs/libra", { :controller => "air", :action => "scales" })
+  # get ("/zodiacs/aquarius", { :controller => "air", :action => "waterbearer" })
+  
+  # get("/zodiacs/cancer " { :controller => "earth" :action => "crab" })
```

Comment:
This is kind of a hard error message to read—
<img width="914" alt="Screen Shot 2020-07-21 at 3 10 10 PM" src="https://user-images.githubusercontent.com/17581658/88101889-50bdb700-cb64-11ea-84c0-a842c0148483.png">

since it's about not inheriting from `ApplicationController` which isn't something we've spent a lot of time talking about.

but I guess it could be a semi common one that they'll run into.

Reply:
@jelaniwoods Yeah I was debating that one, that's why I saved it for the last controller. I think it might be a good opportunity to discuss inheritance and the mechanics of how we're benefitting from Rails?

Reply:
A good debugging checklist in the README that includes "when all else fails, if the error message isn't helping you and you can't spot it, then delete it/comment it out and pave over it" should help get past it.